src/main/java/com/github/manolo8/darkbot/core/objects/facades/DiminishQuestFacade.java

```diff
@@ -0,0 +1,56 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import com.github.manolo8.darkbot.core.objects.Gui;
+import eu.darkbot.api.API;
+import eu.darkbot.api.managers.QuestAPI;
+
+import static com.github.manolo8.darkbot.Main.API;
+
+public class DiminishQuestFacade extends Updatable implements API.Singleton {
```

Comment:
```suggestion
public class DiminishQuestMediator extends Updatable implements API.Singleton {
```
We try to keep naming of mediator and proxy in our classes

Reply:
Done